gateway-messages/tests/versioning/v11.rs

```diff
@@ -36,7 +36,7 @@ fn host_request() {
 
 #[test]
 fn vpd_protocol_errors() {
-    let mut out = [0; MgsRequest::MAX_SIZE];
+    let mut out = [0; SpResponse::MAX_SIZE];
```

Comment:
This is a drive-by fix, because I noticed we're encoding an `SpResponse` below (not an `MgsRequest`)